mxcube3/routes/qutils.py

```diff
@@ -286,6 +286,11 @@ def _handle_dc(sample_id, node):
     parameters.pop('centred_position')
     queueID = node._node_id
     enabled, state = get_node_state(queueID)
+    try:
+        lims_data = mxcube.rest_lims.get_dc(node.id)
```

Comment:
I understand that you might want to handle it in this way, but the real problem is that the mockup is missing a method. So this is something that should not happen, we should add a new mockup. Ill do that as soon as I can since I already said that I would do this.

So ill add a mockup that we can use that have the method.